domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/validation/casefield/CaseFieldEntityValidationContext.java

```diff
@@ -29,4 +33,7 @@ public String getCaseReference() {
         return this.caseReference;
     }
 
+    public List<CaseRoleEntity> getCaseRoles() {
```

Comment:
we should rename this to:
`getCaseTypeCaseRoles` 
it's a bit confusing otherwise